Taskfile.yaml

```diff
@@ -132,6 +133,17 @@ tasks:
       - "{{ .TMP_DIR }}"
     cmd: "mkdir -p {{ .TMP_DIR }}"
 
+  ## Dependency tasks #################################
+
+  update-anchore-dependencies:
+    desc: Update Anchore dependencies
+    deps: [tools]
+    requires:
+      vars:
+        - FROM
+    cmd: |
+      {{ .GH }} workflow run update-anchore-dependencies.yml -f repos="github.com/anchore/stereoscope@{{ .FROM }}"
```

Comment:
Would this task be used for anything? Generally, I think the dependencies would be updated by dependabot once an upstream project has a new version released. If I'm making a change in syft, here, I would be most interested in testing _this specific change_ across the downstream projects, namely: Grype.

Reply:
when there are releases, yes. But this is also used to control integration branches too (update to commits without releases), something that dependabot can't do. This mechanism would happen to be used for both integration and main though during release trains.